.github/workflows/eslint.yml

```diff
@@ -0,0 +1,88 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# ESLint is a tool for identifying and reporting on patterns
+# found in ECMAScript/JavaScript code.
+# More details at https://github.com/eslint/eslint
+# and https://eslint.org
+
+name: ESLint
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '24 21 * * 4'
+
+jobs:
+  eslint_client:
+    name: client eslint
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      security-events: write
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install ESLint
+        working-directory: ./client
+        run: |
+          npm install eslint@8.56.0
+          npm install @microsoft/eslint-formatter-sarif@2.1.7
+          npm install eslint-plugin-react@latest --save-dev
+          npm install @typescript-eslint/eslint-plugin@latest --save-dev --force
```

Comment:
Try to just do npm install.
You're inside the client directory already which has a package file. You should just be able to do an npm install and then run the eslint command. It's fine if other packages are installed.
If you continue to do this then you'd have to manage this workflow file for any change to eslint and that sounds not fun.

Reply:
fixed, i was using a default template and this way makes a lot more sense